### DISCUSSION TITLE
josecadete.net has a trojan?

### DISCUSSION THREAD

**Discussion by Merneith** [Keriew/augustus#1048] [Q&A]
When I click on the download links, it redirects to josecadete.net. Malwarebytes Browser Guard blocks that website and says it has a trojan. This is the message:

<img width="585" alt="josecadetetrojan" src="https://github.com/Keriew/augustus/assets/116155/63ac1c76-be82-45a3-9a07-f85a37b23aa7">

**Comment by crudelios** [comment]
That is clearly a false positive.

That domain is hosted by me and the files it hosts are the direct builds of augustus.

I'll report to Malwarebytes as a false positive to see if they fix the issue!
